crates/orderbook/src/arguments.rs

```diff
@@ -124,6 +124,10 @@ pub struct Arguments {
     /// Enable limit orders. Once the full limit order flow is implemented, this can be removed.
     #[clap(long, env, default_value = "false")]
     pub enable_limit_orders: bool,
+
+    /// Max number of limit orders per user.
+    #[clap(long, env)]
```

Comment:
I think this should have a default value to not cause any surprising issues when deploying it.

Reply:
I think 10 is reasonable. I would err on the side of caution to limit the problems we can encounter in case we forget to correctly set this value at some point.